src/pages/history/index.tsx

```diff
@@ -0,0 +1,52 @@
+import getHistory from "@/api/modules/history/getHistory";
+import PageTemplate from "@/components/PageTemplate";
+import { GetServerSideProps } from "next";
+import Head from "next/head";
+
+export default function History( {
+   history,
+}: {
+  history: string;
+}) {
+  const notFound = <div>History Not Found ...</div>
+
+  if(!history) {
+    return notFound
+  }
+
+  return (
+    <div>
+      <Head>
+        <title>History</title>
+      </Head>
+      <PageTemplate>
+        <div className="min-h-screen">
+          <div className="text-heading-3 pt-12 pl-72 text-left font-inter">History of state</div>
+          <div className="p-72 pt-2 text-[16px]text-justify font-inter whitespace-pre-line">
```

Comment:
We're not using Inter font. We're using Open Sans.
Use a H3 for a heading 3. not a div.
288 padding seems like A LOT (pl-72).. which means you're doing something incorrectly. 
No need to do text-[16px].. You should be using a <p> tag anyway

Gotta learn the difference between padding and margin.
Could also look deeper into semantic HTML
Also read up on why fixed values (16px) is bad in some cases

Reply:
noted ill get right on it